### DISCUSSION TITLE
Conflict when using the same device in 2 AnalogOutput nodes

### DISCUSSION THREAD

**Discussion by PiezaKake** [bonsai-rx/bonsai#2019] [Q&A]
Hi,

I am getting the error below when I run my workflow. Both AnalogOutput nodes are configured to the same device, but different output channels. However, after the delay time, I get this error, possibly because the device is still reserved by the first AnalogOutput node? How can I overcome this?
![image](https://github.com/user-attachments/assets/49a3d2f3-b00f-43eb-b434-cd7300e6b273)

**Comment by bruno-f-cruz** [comment]
Please include the workflow. Also, check if your device allows you to configure two analog output channels asynchronously. At the end of the day Bonsai will call the nidaq API, if the API says its not possible it is likely a problem with the hardware/task configuration being incompatible.
